public/app/plugins/panel/table/cells/ColorBackgroundCellOptionsEditor.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { SelectableValue } from '@grafana/data';
+import {
+  TableCellDisplayMode,
+  TableCellBackgroundDisplayMode,
+  TableColoredBackgroundCellOptions,
+} from '@grafana/schema';
+import { HorizontalGroup, Select, Field } from '@grafana/ui';
+
+import { TableCellEditorProps } from '../models.gen';
+
+const colorBackgroundOpts: SelectableValue[] = [
+  { value: TableCellBackgroundDisplayMode.Basic, label: 'Basic' },
+  { value: TableCellBackgroundDisplayMode.Gradient, label: 'Gradient' },
+];
+
+export const ColorBackgroundCellOptionsEditor = ({
```

Comment:
This component is almost exactly the same as BarGaugeCellOptionsEditor. Would it make sense to refactor this to one component? wdyt?

Reply:
Let me see if I can get merge this in a way that will work out nicely with the types.

Reply:
So I am able to merge these together, however the gauges at least are slated to get more settings after this lands so while they are similar now I think they'll be branching out separately soon. See https://github.com/grafana/grafana/issues/45442

That being said I think I should be able to simplify these components a bit.

Reply:
got it. no worries.